src/Adapter/Security/CommandHandler/BulkDeleteCustomersSessionsHandler.php

```diff
@@ -0,0 +1,59 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+namespace PrestaShop\PrestaShop\Adapter\Security\CommandHandler;
+
+use CustomerSession;
+use PrestaShop\PrestaShop\Core\Domain\Security\Exception\SessionNotFoundException;
+use PrestaShop\PrestaShop\Core\Domain\Security\Command\BulkDeleteCustomersSessionsCommand;
+use PrestaShop\PrestaShop\Core\Domain\Security\CommandHandler\BulkDeleteCustomersSessionsHandlerInterface;
+
+/**
+ * Handles command that deletes customers sessions in bulk action.
+ *
+ * @internal
+ */
+final class BulkDeleteCustomersSessionsHandler implements BulkDeleteCustomersSessionsHandlerInterface
```

Comment:
just noticed in some handlers we use singular form, in others plural.
I wonder if we should start doing all the same & which is the preferred way :thinking: 
1.`BulkDeleteCustomersSessionsHandler`(this also makes you wander between CustomerSessions/CustomersSession/CustomersSessions)
 vs
2.`BulkDeleteCustomerSessionHandler`.
I prefer nr.2 as the"bulkDelete" already explains its usage & then we dont need to choose between multiple plural forms of CustomerSession :man_shrugging:

Reply:
I have no preference here, tell me what to do, and I will :D Should be write in the devdocs to be specify somewhere :)

Reply:
I would suggest using singular form
BulkDeleteCustomerSessionHandler.

Reply:
After starting to do it. A bulk is going to delete a lot of customers' sessions, I think this part needs to be kept in plural mode :thinking:

Reply:
Should be better now ^^